### ISSUE TITLE
Client crash when closing eventlog window

### ISSUE THREAD

**Issue by Bluscream** [pathmann/pyTSon_repository#10]
For some reason my client likes to crash when i close the eventlog window after leaving it open (paused) for some time >~10 mins

Moved from https://github.com/pathmann/pyTSon/issues/102

**Comment by pathmann** [comment]
Does it still occur, if you restrict the events (to 5 for starters)?

**Comment by Bluscream** [comment]
I'm currently testing that.

**Comment by Bluscream** [comment]
Yes, the issue still occurs when using extremely low numbers like 5,1,2 it also occurs when i pause the log right away

**Comment by pathmann** [comment]
Can't reproduce here.

Any output in the log or stdout?

**Comment by Bluscream** [comment]
Not really, you know how debuggable crashes are in TeamSpeak

note that i use Win64
